docs/.vuepress/config.js

```diff
@@ -41,7 +41,7 @@ module.exports = {
         link: "/sponsors/",
       },
     ],
-    repo: "https://github.com/ChristianChiarulli/LunarVim",
+    repo: "https://github.com/LunarVim/LunarVim",
```

Comment:
This is important.  Can you move this to it's own pr and I'll accept it.

Reply:
So to fix this would I `git checkout <this commit> this/file/path`, change it back and push to this branch?